PolyEngine/Standalone/Windows/CoreConfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"DebugNormalsFlag": false,
```

Comment:
Please remove other config related stuf other than asset paths and rename file to AssetPaths.json